src/lib/forms-validation/index.ts

```diff
@@ -272,6 +272,25 @@ function validateWithFormSchema(form?: unknown):
       }
     }
 
+    const labels: string[] = []
```

Comment:
I don't see any code to push items into this array, but perhaps I'm an idiot

Reply:
Harr yep.